src/tcgwars/logic/impl/gen3/HolonPhantoms.groovy

```diff
@@ -2443,20 +2514,21 @@ public enum HolonPhantoms implements LogicCardInfo {
           onAttack {
             damage 20
             afterDamage {
-              flipUntilTails {
-                attachEnergyFrom(my.discard, self)
-              }
+              def n = 0
+              flipUntilTails { n++ }
+              n.times{ attachEnergyFrom(my.discard, self) }
```

Comment:
This should reduce the number of energy selection pop ups shown to only 1. Not sure if the `min` is required, but put it there for safety.
```suggestion
              n = Math.min(n, my.discard.filterByType(BASIC_ENERGY).size())
              attachEnergyFrom(count:n, my.discard, self)
```

Reply:
Sounds good, will put an `if (n)` though just as to avoid a broken select screen (Could handle it in the static, but I'd rather not touch it in this PR)

Reply:
I have to ask, what is an EE?

Reply:
Engine Error.

Reply:
Okay. Since that isn't a standard error type shortening it like that is very confusing for anybody who is seeing it for the first time.

Reply:
Yup, been using it a lot in commits so just shortened, sorry if it wasn't very clear 😛